src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ProfessionalExternalUserFunctionalTest.java

```diff
@@ -67,12 +69,16 @@ public void testExternalUserScenario() {
         findUserStatusByEmailScenarios();
         modifyRolesScenarios();
         suspendUserScenarios();
+        findMFAScenario();
     }
 
-    public void setUpTestData() {
+    public void setUpOrgTestData() {
```

Comment:
make sure this method is called only once for mfa and other test cases otherwise it will mess u things

Reply:
We have added @Test annotation also for the mfa test scenarios now, will it still be a problem as they will be considered different tests?